mtoken-model/src/main/java/com/wultra/security/powerauth/lib/mtoken/model/request/OperationApproveRequest.java

```diff
@@ -33,4 +34,8 @@ public class OperationApproveRequest {
     @NotNull
     private String data;
 
+    /**
+     * Optional OTP used for proximity check. User is instructed by {@link PreApprovalScreen.ScreenType#QR_SCAN}.
+     */
+    private String otp;
```

Comment:
Can we use a more descriptive variable name than `otp`? There are several different OTPs involved with PowerAuth operations.

Reply:
Good idea, renamed to `proximityCheckOtp`.